templates/ad_tech_platform.yaml

```diff
@@ -0,0 +1,4 @@
+---
+template:
+  name: adTechPlatform
```

Comment:
are we doing snake case or camel case in general?

Reply:
Right now the repo isn't clear actually. I'm see both. Need to pick one and then change it and be consistent moving forward

Reply:
PEP8 recommends snake case `_` for everything, i.e. function name, variables etc. So we should update the code if we want to conform with that style guide.